content/doc/book/system-administration/administering-jenkins-on-kubernetes.adoc

```diff
@@ -12,4 +12,91 @@ ifdef::backend-html5[]
 :toc: left
 endif::[]
 
-= Administering Jenkins on Kubernetes
\ No newline at end of file
+= Administering Jenkins on Kubernetes
+
+In this section, we will look at:
+
+* Upgrading from one version of Kubernetes to another.
+* How to upgrade the version of Kubernetes in your cluster.
+* Upgrade Kubernetes cluster maintained by the Jenkins Infrastructure project.
```

Comment:
I'm sorry to drop in without having been part of the preliminaries phases but I have some difficulties to understand this content.

Why would you want to upgrade a Kubernetes cluster in regards to installing Jenkins?
As much as the section about Kubernetes concepts looks good by pointing to external resources, I feel the preliminaries here are not really helping a Kubernetes admin.

As a Kubernetes admin, I would start by looking at the installation method(s). In our case, the Jenkins Helm Chart is clearly the de-factor method: other methods are edge cases or not really common.

The prerequisites are not really related to the kubernetes version but more to storage (as a Jenkins constraint) and ingress (to access Jenkins).

What do you think?